minikube_demo_setup.sh

```diff
@@ -44,6 +49,32 @@ function check_err() {
 	fi
 }
 
+# Prints the minimum system resources required to run the demo
+function print_min_resources() {
+  echo "       Minikube resource config needed for demo:"
+  echo "       CPUs=8, Memory=16384MB"
+}
+
+# Checks if the system which tries to run autotune is having minimum resources required
+function sys_cpu_mem_check() {
+  SYS_CPU=$(lscpu -p | grep -v '^#' | wc -l)
+  SYS_MEM=$(grep MemTotal /proc/meminfo | awk '{printf ("%.0f\n", $2/(1024))}')
+
+  if [ "$SYS_CPU" -lt "$MIN_CPU" ]; then
```

Comment:
Best practice is to use quotes and brackets

```
 "${SYS_CPU}"      "${MIN_CPU}"
```
Please make this change everywhere

Reply:
@dinogun I have made the changes